1-getting-started.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
_Commented on notebook `1-getting-started.ipynb` Cell 88 Line 9_
```markdown

You can install jupyter lab by running: 

`$conda install jupyter lab` from the Anaconda Prompt CLI

Note, jupyter lab and notebook were already installed in the `0-pre-course-assignment` module.

### Basic Commands to Launch Jupyter Tools
```
Are we gonna just use VS Code to do our notebooks in this course? If so, maybe move this section to be after VS Code, as a FYI bit.